kolibri_instant_schools_plugin/assets/src/views/sign-in-page/index.vue

```diff
@@ -275,10 +302,11 @@
         this.showDropdown = false;
       },
       signIn() {
+        const strippedPhoneNumber = this.username.replace(/\D/g, '');
         this.formSubmitted = true;
         if (this.formIsValid) {
           this.kolibriLogin({
-            username: this.username,
+            username: strippedPhoneNumber,
```

Comment:
I thought this was handled by the server @jamalex? Didn't seem to be working locally, so I put this in